doc/decisions/README.md

```diff
@@ -55,6 +55,12 @@ section here.
 
 - [Array for Warnings](warning_array.md)
 - [Array](array.md)
+- [Error Handling](error_handling.md)
+- [Iterators](iterators.md)
+- [keyString() return value](key_string_return_value.md)
```

Comment:
As this depends on error_handling.md, probably we should also move it to "In Discussion".